56.md

```diff
@@ -2,35 +2,37 @@
 NIP-56
 ======
 
-Reporting
+Labeling
 ---------
 
-`draft` `optional` `author:jb55`
+`draft` `optional` `author:jb55` `author:michaelhall923` `author:gruruya`
 
-A report is a `kind 1984` note that is used to report other notes for spam,
-illegal and explicit content.
+A label is a `kind 1984` note that describes other notes or pubkeys.  
 
 The content MAY contain additional information submitted by the entity
-reporting the content.
+labeling the content.
 
 Tags
 ----
 
-The report event MUST include a `p` tag referencing the pubkey of the user you
-are reporting.
+The label event contains one or more tags:
+- `[e, <event-id>, <label>]` - Label event
+- `[p, <pubkey>]` - Use pubkey to find recommended relays
+- `[p, <pubkey>, <label>]` - Same as above, but also label pubkey
```

Comment:
Labeling a pubkey essentially means labeling the relay which are found using this pubkey, right?

I would make this more clear then.

Reply:
No, it's the same as reporting the pubkey was so it's labeling the account/profile. I haven't detailed a way to label relays (yet?)

Reply:
Ah, oh, I see. I didn't read how it was before.

Reply:
Is it clear the way I ordered it?
https://github.com/nostr-protocol/nips/blob/8e657d0e38dda371217188502e27c0e91f48882a/56.md?plain=1#L21

Reply:
Labeling relays probably isn't a good idea because ip addresses and domain names change. There is no universal pointer to a relay like there is pubkeys for events/profiles. This seems like a better use of NIP-11 https://github.com/nostr-protocol/nips/blob/master/11.md

Reply:
Really good point.